testgrid/config.yaml

```diff
@@ -292,6 +292,16 @@ test_groups:
   days_of_results: 60
   num_columns_recent: 3
   num_failures_to_alert: 1
+- name: ci-kubernetes-e2e-gce-scale-coredns
+  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-scale-coredns
+  days_of_results: 60
```

Comment:
This and the settings below are not really much relevant for your jobs (we had them in our large-scale jobs for different reasons that don't really apply for you I think).